docs/services/postgresql/add-standby-node.md

```diff
@@ -32,6 +32,84 @@ $ cchq <env> ap setup_pg_standby.yml -e standby=[standby node]
 
 
 # Promoting a hot standby to Master
+1. Begin downtime for your site: `$ commcare-cloud <env> downtime start`
+
+2. Verify that the replication is up to date
+
+```
+$ commcare-cloud <env> run-shell-command pg1,pg2 'ps -ef | grep -E "sender|receiver"'
```

Comment:
maybe make a note that `pg1,pg2` is the list of postgres nodes as defined in your inventory file

Reply:
https://github.com/dimagi/commcare-cloud/pull/3072/commits/2292bf75e85799502b9fc1cadd696e254d609722 Created `pg_database` and `pg_standby`